src/ui/AccountStatus.svelte

```diff
@@ -1,21 +1,57 @@
 <script>
-  import Svg from '@/ui/Svg/svelte'
-  import { getSanbaseSubscription, getTrialDaysLeft } from '@/utils/subscription'
+  import { customerData$ } from '@/stores/user'
+  import { subscription$ } from '@/stores/subscription'
+  import { PlanName } from '@/utils/plans'
+  import { AccountDropdown } from '@/utils/enums'
 
   export let currentUser
+  export let variant = AccountDropdown.Version1
 
-  $: subscription = currentUser && getSanbaseSubscription(currentUser.subscriptions)
-  $: trialDaysLeft = subscription && getTrialDaysLeft(subscription)
-  $: trialPostfix = trialDaysLeft ? ` Trial (${trialDaysLeft})` : ''
+  $: ({ isEligibleForTrial, annualDiscount } = $customerData$)
+  $: trialSpecialOfferPercent =
+    (annualDiscount &&
+      annualDiscount.isEligible &&
+      annualDiscount.discount &&
+      annualDiscount.discount.percentOff) ||
+    0
```

Comment:
`trialSpecialOfferPercent` is a misleading name of the variable, since the `annualDiscount` is not connected to the `trial` status.
`annualDiscount` name is good enough.

Reply:
We already have `annualDiscount` variable